.travis.yml

```diff
@@ -9,42 +9,80 @@
 ##
 
 sudo: required
+
 dist: trusty
+
 branches:
   only:
     - master
-matrix:
-  fast_finish: true
 
-env:
-  - TARGET="Style"
-  - TARGET="Registers"
+matrix:
+  include:
+    - os: osx
```

Comment:
Testing all these versions of Xcode is not super useful. Having only 7 and 7.3 is good enough for now probably.